inter-ident-verif/README.md

```diff
@@ -0,0 +1,62 @@
+# Interpersonal Identity Verification
```

Comment:
The file should be in designs/inter-ident-verify.md not inter-ident-verify/readme.md :)